server/db/models/index.js

```diff
@@ -1,4 +1,5 @@
 const User = require('./user')
+const Products = require('./products');
```

Comment:
const Product = require('./product');
 
also rename the whole file product.js and rename the spec files product.spec.js